RHEL/6/input/profiles/C2S.xml

```diff
@@ -512,7 +512,7 @@ baseline.
 <select idref="sshd_allow_only_protocol2" selected="true" />
 
 <!-- CIS 6.2.2 Set LogLevel to INFO (Scored) -->
-<!-- Default, non-configurable system behavior, will audit user login/logout events -->
+<select idref="sshd_set_loglevel_info" selected="true" />
```

Comment:
I think the XCCDF is missing for RHEL6, it does not seem to use the shared ssh.xml (https://github.com/hex2a/scap-security-guide/blob/d30a94405411767df6e0a9b175d638854b242eb7/RHEL/6/input/guide.xslt#L134)